src/mappings/content/channel.ts

```diff
@@ -368,10 +358,8 @@ export async function processChannelFundsWithdrawnEvent({
   await addNotification(
     overlay.getEm(),
     channelOwnerAccount,
-    new ChannelFundsWithdrawn({
-      recipient: new ChannelRecipient({ channelTitle: parseChannelTitle(channel) }),
-      amount,
-    }),
+    new ChannelRecipient({ channel: channel.id }),
+    new ChannelFundsWithdrawn({ amount }),
```

Comment:
According to https://github.com/Joystream/atlas/issues/4891#issuecomment-1725299795 this notification doesn't appear (I'll check tomorrow whether it comes from Orion).